optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/report/BenchmarkReport.java

```diff
@@ -529,6 +549,64 @@ private void writeBestScorePerTimeSpentSummaryChart() {
         }
     }
 
+    private void writeSubSingleBenchmarkScoreCharts() {
```

Comment:
Thinking aloud: Does this method belong on this BenchmarkReport?
It only had summary charts. These subsingle score charts are per problem I guess? But it doesn't hurt to generate it all the time (iff subSingleCount > 1 of course) as it doesn't impact the solver speed. So it is kinda like a summary chart and it's not... Let's keep it here for now.

Will these charts be made obsolete by the summary version of this chart:
  https://issues.jboss.org/browse/PLANNER-443
Or does that summary version has less detailed information? Let's worry about that later (we can easily delete this one if needed).